backend/shared/src/websockets/helpers.ts

```diff
@@ -90,8 +90,12 @@ export function broadcastNewAnswer(answer: Answer) {
 }
 
 export function broadcastUpdatedAnswer(answer: Answer) {
-  const payload = { answer }
-  const topics = [`contract/${answer.contractId}/updated-answer`]
+  broadcastUpdatedAnswers([answer])
+}
+
+export function broadcastUpdatedAnswers(answers: Answer[]) {
```

Comment:
This is good, however, to make it useful, you need to refactor `place-bet.ts` and other places that use updateAnswer to send the whole array of updated answers at once.